src/functions/wceStyles.ts

```diff
@@ -367,11 +367,11 @@ export default function wceStyles(): void {
   }
   #wce-chat-baby-talk[data-state="ignore"]::before,
   #wce-chat-stutters[data-state="ignore"]::before {
-    background-color: #bbb;
+    --button-color: #bbb;
```

Comment:
This is inside a template literal, so you could so something like:

```suggestion
    ${GameVersion === "R104" ? "background" : "--button"}-color: #bbb;
```

Reply:
Oh, neat! Wasn't 100% certain of BC symbols could already be accessed at this point, but this does the trick nicely in that case.

Applied as of https://github.com/KittenApps/WCE/pull/29/commits/7125374c0e755734d7e55e86ceacb3829d20d5b5

Reply:
`GameVersion ` is available very early, even before you login, which was important here

Thanks for the fix.